### PR TITLE
Add docs for udpated event

### PR DESCRIPTION
**Pull request by dubeyShivank**
Only to be merged in after v0.1.46 is released

### CODE REVIEW

File: packages/millicast-sdk/src/types/index.d.ts [hunk 1]
```diff
@@ -714,7 +714,7 @@ declare module '@millicast/sdk' {
      */
     sourceId?: string;
     /**
-     * - Override which events will be delivered by the server ("active" | "inactive").
+     * - Override which events will be delivered by the server ("active" | "inactive" ).
```
Comment (Santiago-Souto) [inline packages/millicast-sdk/src/types/index.d.ts:717]: Is this change correct?

Comment (dubeyShivank) [reply packages/millicast-sdk/src/types/index.d.ts:717]: Na, I'm sure I added in some text there and removed it lol. Will fix.

Comment (dubeyShivank) [reply packages/millicast-sdk/src/types/index.d.ts:717]:
```suggestion
     * - Override which events will be delivered by the server ("active" | "inactive").
```

**Review by Santiago-Souto — APPROVED** [review]
Looks good! Should we wait for media server team to also approve?

**Comment by dubeyShivank** [comment]
> Looks good! Should we wait for media server team to also approve?

Yes, if @mildsunrise can approve that'll be excellent :)

**Review by mildsunrise — APPROVED** [review]
sorry, been really busy this week, lgtm

**Pull request merged**